### PR TITLE
fix test flake when kubectl e2e and crd-openapi e2e run in parallel

### PR DESCRIPTION
**Pull request by roycaihw**
**What type of PR is this?**
/kind failing-test

**What this PR does / why we need it**:
Fixes test flake. [kubectl e2e test](https://github.com/kubernetes/kubernetes/blob/master/test/e2e/kubectl/kubectl.go) may fail if it runs in parallel with [crd openapi e2e test](https://github.com/kubernetes/kubernetes/blob/master/test/e2e/apimachinery/crd_publish_openapi.go): 

https://storage.googleapis.com/k8s-gubernator/triage/index.html?text=array%20should%20have%20exactly%20one%20sub-item&test=%5C%5Bsig%5C-cli%5C%5D%5C%20Kubectl

**Which issue(s) this PR fixes**:

mitigates https://github.com/kubernetes/kubernetes/pull/78234#issuecomment-495821109

**Does this PR introduce a user-facing change?**:

```release-note
NONE
```

/cc @sttts @liggitt

### CODE REVIEW

File: test/e2e/apimachinery/crd_publish_openapi.go [hunk 1]
```diff
@@ -598,4 +598,4 @@ properties:
     properties:
       bars:
         description: List of Bars and their statuses.
-        type: array`)
+        type: object`)
```
Comment (liggitt) [inline test/e2e/apimachinery/crd_publish_openapi.go]: would it be better to define `items: {type: object}`?

Comment (roycaihw) [reply test/e2e/apimachinery/crd_publish_openapi.go]: that's better. Updated

**Comment by roycaihw** [comment]
fixing `pull-kubernetes-kubemark-e2e-gce-big` failure in https://github.com/kubernetes/perf-tests/pull/541

**Pull request merged**